templates/usr/local/kafka/config/log4j.properties.j2

```diff
@@ -22,7 +22,6 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
 
 log4j.appender.kafkaAppender=org.apache.log4j.RollingFileAppender
-log4j.appender.kafkaAppender.DatePattern='.'yyyy-MM-dd-HH
```

Comment:
Oh interesting, because I think I put that there after referencing the log4j docs.  Reviewing my dev and prod environments I can definitely concur that the datePattern is not applied.  LGTM.